src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -59,13 +60,24 @@ export interface TsvUploadResponse {
   recordsModified: number;
 }
 
+export interface RecordResponseBody {
+  id: string;
+  type: string;
+  attributes: object;
```

Comment:
Nit. This type could be slightly more descriptive as something like:

```suggestion
  attributes: { [attributeName: string]: any };
```